frontend/src/main/scala/bloop/util/CommandsDocGenerator.scala

```diff
@@ -7,43 +7,49 @@ import bloop.Cli
 import bloop.cli.{CliParsers, Commands, CommonOptions}
 import bloop.engine.Run
 import caseapp.{Name, ValueDescription}
-import caseapp.core.{Arg, Messages, NameOps, ValueDescriptionOps}
+import caseapp.core.Arg
+import caseapp.core.help.Help
+import caseapp.core.util.NameOps._
 
 import scala.util.control.NonFatal
 import bloop.io.Environment.lineSeparator
+import caseapp.core.util.Formatter
 
 object CommandsDocGenerator {
   private final val docsContentDelimiter =
     "<!-- This file was generated by bloop.util.CommandsDocGenerator (see developer docs) -->"
+  private val NL = System.lineSeparator()
```

Comment:
Best to use `bloop.io.Environment.lineSeparator` which makes sure that even if we are on Windows and use for example ZSH, we will just get `\n`.

Reply:
Good point, just fixed that!